backend/src/main/java/net/gunn/elimination/model/EliminationUser.java

```diff
@@ -177,4 +178,25 @@ public String toString() {
     public boolean equals(Object obj) {
         return obj instanceof EliminationUser && ((EliminationUser) obj).getSubject().equals(getSubject());
     }
+
+	// idfk what to cll thism ethod
+	public Map decompose() {
```

Comment:
u could make a class called PublicUserSerialiser and then annotate elimuser with @JsonSerialise(using = ...)